pkgs/tools/networking/edgedb/default.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenv
+, lib
+, runCommand
+, patchelf
+, fetchFromGitHub
+, rustPlatform
+, makeWrapper
+, pkg-config
+, curl
+, Security
+, CoreServices
+, libiconv
+, xz
+, perl
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "edgedb";
+  version = "1.2.0-unstable";
```

Comment:
First, I have seen no 1.2.0 release, only a 1.1.2 on https://github.com/edgedb/edgedb-cli/tags

Second, if your intention was using the bleeding master, you should write this as `unstable-<YYYY-MM-DD>`, where the date is from the commit you want to include.

Third, the commit message should have the format

`edgedb: init at <version>`

Reply:
Indeed, the v1.2.0 is the next version, currently in development on master